.studio/compliance-service

```diff
@@ -95,6 +95,16 @@ function load_compliance_reports() {
     pushd components/compliance-service/test_data/audit_reports &> /dev/null
     ./send_to_data_collector.sh https://a2-dev.test "$(get_admin_token)"
     popd &> /dev/null
+    generate_chef_run_example | jq '.entity_uuid = "34cbbb4c-c502-4971-b193-00e987b4678c"' | jq '.node_name = "debian(2)-zeta-linux(f)-apache(p)-failed"' | send_chef_data_raw
```

Comment:
i considered trying to read the id and name off each file in the audit reports folder, but it seemed like overkill